src/service/errorDecoder/providers/entrypoint/decoders/AA21Decoder.ts

```diff
@@ -3,23 +3,33 @@ import { networkConfig } from "../../../../../config/network";
 import { EntryPointFactory } from "../../../../../repository/entryPoint/factory";
 import { DecodedError, ErrorSource } from "../../../../../types";
 import { ErrorDecoderParams, IErrorDecoder } from "../../../interface/IErrorDecoder";
+import { trim } from "../../../../../utils";
 
 export class AA21Decoder implements IErrorDecoder {
+
     async decodeError(param: ErrorDecoderParams): Promise<DecodedError> {
         // TODO: Extract more information that is related to this error eg., the amount of native balance the smart account has
         // and how much is needed to pay for the gas.
         let epService = EntryPointFactory.getEntryPointService(param.entryPointAddress);
         let requiredFunds;
+        let symbol, suggestedActions: string[] = [];
```

Comment:
if these variables are not reused, should use const instead.

Reply:
These variables are initialised inside the if statement. Making them const is giving the error.